bsp_diff/caas/device/intel/civ/host/vm-manager/0006-Revert-Upgrade-the-qemu4.2-to-6.0-and-add-SRIOV-chan.patch

```diff
@@ -0,0 +1,132 @@
+From 88748b684640a8e45ce95d1ebc4bf3ae59d7ed52 Mon Sep 17 00:00:00 2001
+From: "Mishra, ManishX" <manishx.mishra@intel.com>
+Date: Mon, 23 May 2022 16:23:37 +0530
+Subject: [PATCH] Revert "Upgrade the qemu4.2 to 6.0 and add SRIOV changes"
+
+This reverts commit 981fde7afd133b128e831695b9aa983dbc71b8dc.
+---
+ scripts/setup_host.sh      | 37 ++++++-------------------------------
+ scripts/start_flash_usb.sh |  2 +-
+ 2 files changed, 7 insertions(+), 32 deletions(-)
+
+diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
+index 57d55ed..667a37c 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -9,7 +9,7 @@ set -eE
+ 
+ #---------      Global variable     -------------------
+ reboot_required=0
+-QEMU_REL="qemu-6.0.0"
```

Comment:
The qemu 6.0 change is introduced by 0002-Switch-CIV-to-use-QEMU-6.0.0.patch. This patch is for ADL and should not be picked up by tpy branch. I checked in TPY branch, the qemu version is still 4.2